addons/servicegraph/docker/Dockerfile.servicegraph_debug_ppc64le

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:xenial
```

Comment:
Rather than adding a new docker file an alternative is to make the existing file more customizable.  E.g. the original file would have something like:
<pre>
- FROM gcr.io/istio-testing/ubuntu_xenial_debug:3f57ae2aceef79e4000fb07ec850bbf4bce811e6f81dc8cfd970e16cdf33e622
+ UBUNTU_DEBUG_IMAGE=gcr.io/istio-testing/ubuntu_xenial_debug:3f57ae2aceef79e4000fb07ec850bbf4bce811e6f81dc8cfd970e16cdf33e622
+ FROM ${UBUNTU_DEBUG_IMAGE}
</pre>
and then the Makefile would have something like:
<pre>
ifeq ($(MACH), ppc64le)
DOCKER_OPT+=--build-arg UBUNTU_DEBUG_IMAGE=ubuntu:xenial
endif
</pre>
and then add a $DOCKER_OPT after "docker build" in the DOCKER_RULE.

This proposal isn't perfect since the "FROM" for Linux vs PPC would live in two different places but it would help to avoid the duplication of files and reduce the severity of the changes in the makefile.  Since an actual debug image of xenial isn't being used I'm not sure if some one argue that it's be better to just not make the debug images part of a build on ppc.

I'm not sure what to do about grafana since it seems to differ more than just the from line.

Reply:
I'll show my lack of docker knowledge but isn't ubuntu:xenial a linux x86 image ?
(oh do you make non x86 images in docker?)

Reply:
> Rather than adding a new docker file an alternative is to make the existing file more customizable.

@mattdelco yeah, this makes more sense. Will change this to a customizable base image. grafana has the same case too, i.e., it just differs w.r.t the base image.

>  isn't ubuntu:xenial a linux x86 image ? oh do you make non x86 images in docker?

@ldemailly No and yes. docker.io registry supports multi-arch manifest for some of the images. 
`ubuntu:xenial` is one of them. So, it basically depends from which architecture we are pulling the image from and will pull the appropriate image.